tools/docker/docker-compose-stage.yaml

```diff
@@ -201,6 +179,15 @@ services:
     ports:
       - '${EDA_PROXY_PORT:-3128}:3128'
 
+  eda-ui:
+    image: ${EDA_UI_IMAGE:-quay.io/ansible/eda-ui}:${EDA_UI_VERSION:-main}
+    environment: *common-env
+    ports:
+      - '${EDA_UI_PORT:-8443}:443'
+    depends_on:
```

Comment:
Do we really need the conditional for the UI? The UI is agnostic and it is not really dependent on the api.

Reply:
@Alex-Izquierdo I have seen that the nginx embedded inside the UI image tries to reach out and fails. Not sure if that is when you have an idling browser window that tries to send in the /me call when the system is just coming back up.